sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/rx/FullTextIndexTest.java

```diff
@@ -0,0 +1,154 @@
+import com.azure.cosmos.models.PartitionKeyDefinition;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Ignore;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.fail;
+
+@Ignore("TODO: Ignore these test cases until the public emulator with vector indexes is released.")
+public class FullTextIndexTest extends TestSuiteBase{
+    protected static final int TIMEOUT = 30000;
+    protected static final int SETUP_TIMEOUT = 20000;
+    protected static final int SHUTDOWN_TIMEOUT = 20000;
+
+    protected static Logger logger = LoggerFactory.getLogger(VectorIndexTest.class.getSimpleName());
+    private final ObjectMapper simpleObjectMapper = Utils.getSimpleObjectMapper();
+    private final String databaseId = CosmosDatabaseForTest.generateId();
+    private CosmosAsyncClient client;
+    private CosmosAsyncDatabase database;
+
+    @BeforeClass(groups = {"emulator"}, timeOut = SETUP_TIMEOUT)
+    public void before_VectorIndexTest() {
+        // set up the client
+        client = new CosmosClientBuilder()
+            .endpoint(TestConfigurations.HOST)
+            .key(TestConfigurations.MASTER_KEY)
+            .directMode(DirectConnectionConfig.getDefaultConfig())
+            .consistencyLevel(ConsistencyLevel.SESSION)
+            .contentResponseOnWriteEnabled(true)
+            .buildAsyncClient();
+
+        database = createDatabase(client, databaseId);
+    }
+
+    @AfterClass(groups = {"emulator"}, timeOut = SHUTDOWN_TIMEOUT, alwaysRun = true)
+    public void afterClass() {
+        safeDeleteDatabase(database);
+        safeClose(client);
+    }
+
+    @Test(groups = {"emulator"}, timeOut = TIMEOUT*10000)
+    public void shouldCreateFullTextPolicy() throws Exception {
+        PartitionKeyDefinition partitionKeyDef = new PartitionKeyDefinition();
+        ArrayList<String> paths = new ArrayList<String>();
+        paths.add("/mypk");
+        partitionKeyDef.setPaths(paths);
+
+        CosmosContainerProperties collectionDefinition = new CosmosContainerProperties(UUID.randomUUID().toString(), partitionKeyDef);
+
+        IndexingPolicy indexingPolicy = new IndexingPolicy();
+        indexingPolicy.setIndexingMode(IndexingMode.CONSISTENT);
+        ExcludedPath excludedPath = new ExcludedPath("/*");
+        indexingPolicy.setExcludedPaths(Collections.singletonList(excludedPath));
+
+        IncludedPath includedPath1 = new IncludedPath("/name/?");
+        IncludedPath includedPath2 = new IncludedPath("/description/?");
+        indexingPolicy.setIncludedPaths(ImmutableList.of(includedPath1, includedPath2));
```

Comment:
Create a helper method to eliminate some duplication of this boilerplate in the below test?